ckanext/data_qld/pylons_plugin.py

```diff
@@ -22,7 +22,11 @@ def before_map(self, m):
         m.connect(
             'data_qld.show_schema',
             '/dataset/{dataset_id}/resource/{resource_id}/%s/show/' % constants.SCHEMA_MAIN_PATH,
-            controller=controller, action='show_schema', conditions=dict(method=['GET']))
+            controller=controller, action='show_resource_schema', conditions=dict(method=['GET']))
+        m.connect(
+            'data_qld.show_schema',
```

Comment:
This should use a unique route name, eg 'data_qld.show_dataset_schema'.

Reply:
+

Reply:
Will be fixed and pushed by Alex shortly.